redash/models.py

```diff
@@ -866,6 +871,7 @@ class Query(ChangeTrackingMixin, TimestampMixin, BelongsToOrgMixin, db.Model):
     schedule = Column(db.String(10), nullable=True)
     schedule_failures = Column(db.Integer, default=0)
     schedule_until = Column(db.DateTime(True), nullable=True)
+    schedule_keep_results = Column(db.Integer, nullable=True)
```

Comment:
I'd suggest to use something more descriptive e.g. `schedule_resultset_size` since `schedule_keep_results` is ambiguous whether it's a boolean or integer. That makes the code that uses it harder to read.